apis/core/bootstrap/shapes/dataset.ts

```diff
@@ -384,6 +424,82 @@ ${shapeId} {
       ${sh.message} "Metadata must not have the dcterms:identifier property. Please remove it in advanced RDF editor" ;
       ${sh.group} ${mainGroup} ;
     ] ;
+    ${sh.property} [
+      ${sh.name} "Resources" ;
+      ${sh.path} ${dcat.distribution} ;
+      ${sh.class} ${dcat.Distribution} ;
+      ${sh.nodeKind} ${sh.BlankNode} ;
+      ${dash.editor} ${dash.DetailsEditor} ;
+      ${sh.node} ${opendataSwissResources} ;
+      ${sh.order} 10 ;
+      ${sh.group} ${opendataResourcesGroup} ;
+    ] ;
+  .
+
+  ${opendataSwissResources} a ${sh.NodeShape} ;
+    ${sh.property} [
+      ${sh.name} "Access URL" ;
+      ${sh.path} ${dcat.accessURL} ;
+      ${sh.minCount} 1 ;
+      ${sh.nodeKind} ${sh.IRI} ;
+      ${sh.order} 10 ;
```

Comment:
consider adding a specific validation message like: `${sh.message} "Need some URL to access the data." ;`

The current generic message: 
> Access URL:  Less than 1 values

may not be clear enough when a value is specified but it's not recognized as URL